### PR TITLE
Feat/new style

### PR DESCRIPTION
**Pull request by HenriPro**
New colors and html/css changes to make use consistent.

Screen Shot 2019-09-16 at 10 23 18 AM

### CODE REVIEW

File: workspaces/default/themes/light-theme/assets/styles/site.css [hunk 1]
```diff
@@ -519,12 +535,12 @@ a.tag-link.is-active {
 }
 
 .team-member .member-name {
-  color: var(--grey_900);
+  color: var(--aacent);
```
Comment (nijikokun) [inline workspaces/default/themes/light-theme/assets/styles/site.css]: `aacent` -> `accent`

File: workspaces/default/themes/light-theme/assets/styles/site.css [hunk 2]
```diff
@@ -519,12 +535,12 @@ a.tag-link.is-active {
...
   margin-bottom: 0;
 }
 
 .team-member .member-title {
-  color: var(--grey_700);
+  color: var(--aacent);
```
Comment (nijikokun) [inline workspaces/default/themes/light-theme/assets/styles/site.css]: `aacent` -> `accent`

File: workspaces/default/themes/light-theme/assets/styles/site.css [hunk 3]
```diff
@@ -446,7 +456,7 @@ a.tag-link.is-active {
   padding: 3rem;
   align-items: start;
   background: rgba(255,255,255);
-  box-shadow: 0 1px 2px rgba(0,0,0,0.1), 0 0 10px rgba(0,0,0,0.08);
+  box-shadow: 0 1px 2px var(--border, grey);
```
Comment (nijikokun) [inline workspaces/default/themes/light-theme/assets/styles/site.css]: Wondering if this should stay as a drop shadow, cc @jitachi

File: workspaces/default/themes/light-theme/assets/styles/site.css [hunk 4]
```diff
@@ -274,12 +282,12 @@ input.has-error {
...
     width: 100%;
     background: white;
-    color: var(--accent_500, blue);
+    color: var(accent, blue);
     display: block;
   }
 
   .site-header nav li ul li a:not(.nav-button):hover {
-    background: var(--accent_100, #f1f1f1);
+    background: var(accent, #f1f1f1);
```
Comment (nijikokun) [inline workspaces/default/themes/light-theme/assets/styles/site.css]: `aacent` -> `--accent`

File: workspaces/default/themes/light-theme/assets/styles/site.css [hunk 5]
```diff
@@ -274,12 +282,12 @@ input.has-error {
     padding: .5em 1em;
     width: 100%;
     background: white;
-    color: var(--accent_500, blue);
+    color: var(accent, blue);
```
Comment (nijikokun) [inline workspaces/default/themes/light-theme/assets/styles/site.css]: `accent` -> `--accent`

File: workspaces/default/themes/light-theme/assets/styles/site.css (hunk 3 above)
Comment (jitachi) [reply workspaces/default/themes/light-theme/assets/styles/site.css]: Let's say yes. 🤔

Comment (nijikokun) [reply workspaces/default/themes/light-theme/assets/styles/site.css]: Apologies, realized that my comment wasn't very clear.

Right now it's a solid drop-shadow that's very heavy handed, perhaps it should stay as an reduced alpha channel of black.

**Pull request merged**